src/components/Navbar.tsx

```diff
@@ -1,4 +1,12 @@
-export const Navbar = () => {
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation: React.FC = () => {
+  const getLinkClass = ({ isActive }: { isActive: boolean }) => classNames(
+    'navbar-item',
+    { 'has-background-grey-lighter': isActive },
+  );
```

Comment:
It is better to define this function outside the component so that it is not created every time a new render is performed.